### PR TITLE
Feat/mongo connection

### PR DESCRIPTION
**Pull request by Adi-tya93**
@mila-mamat @atsushii @ylee297 
Can anyone just review this quickly I know its important step so needs to be merged soon

### CODE REVIEW

File: server/middleware/database.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
+const MongoClient = require("mongodb").MongoClient;
+require("dotenv").config;
```
Comment (atsushii) [inline server/middleware/database.js]: Can you try this→ require("dotenv/config");

File: server/middleware/database.js [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+const database = "calendly";
+const collections = "test";
+const uri =
+  "mongodb+srv://team-lavender:lavender-team@cluster0.kenuz.mongodb.net/calendly?retryWrites=true&w=majority";
+// put the URI in process.env.ATLAS_URI, but not working for some reason
+const client = new MongoClient(uri, {
```
Comment (atsushii) [inline server/middleware/database.js]: if it doesn't work, how about using mongoose like below
"""
mongoose
  .connect(process.env.<VARIABLE NAME>, { useNewUrlParser: true })
  .then(() => console.log("successfuly"))
  .catch((err) => console.log(err));
"""

**Review by atsushii — COMMENTED** [review]
If it doesn't work, Can you try to use mongoose?

File: server/.env [hunk 3]
```diff
@@ -1,2 +1,3 @@
 
 TEAM_MEMBERS="Aecio, Bonnie, Shums Kassam, Younhee Lee, Aditya Sharma,Mila Mamat"
+ATLAS_URI="mongodb+srv://team-lavender:lavender-team@cluster0.kenuz.mongodb.net/sample_airbnb?retryWrites=true&w=majority"
```
Comment (bonnieli) [inline server/.env]: put this in an env variable and share the uri with your team

**Pull request merged**